File: Java/Product.java
public class Product {

    /* Private Attribute */
    private String price;
    private String idProduct;

    /* Constuctor */
    public Product(String price, String idProduct) {
        this.price = price;
        this.idProduct = idProduct;
    }

    public Product() {

    }

    /* Setter dan Getter Price */
    public void setPrice(String price) {
        this.price = price;
    }

    public String getPrice() {
        return price;
    }

    /* Setter dan Getter Id Product */
    public void setIdProduct(String idProduct) {
        this.idProduct = idProduct;
    }

    public String getIdProduct() {
        return idProduct;
    }

}

File: Python/Product.py
class Product():

    #private attributes
    __price = "Price"
    __idProduct = "Id Product"

    #Constructor
    def __init__(self, price = "Price", idProduct = "Id Product"):
        self.__price = price
        self.__idProduct = idProduct
    
    #Setter dan Getter Price
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

    #Setter dan Getter Id Product
    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    def getIdProduct(self):
        return self.__idProduct